### ISSUE TITLE
Many debug messages in output

### ISSUE THREAD

**Issue by marcelnem** [angular/vscode-ng-language-service#147]
Messages like these keep appearing in the output window with Angular language service 0.1.4. on Windows 10. It can be distracting.

getDiagnostics: 15ms
getCompletions: 9ms
getDiagnostics: 10ms
getCompletions: 10ms
getCompletions: 15ms
getCompletions: 9ms
getCompletions: 10ms
getCompletions: 8ms
getDiagnostics: 5ms

**Comment by acoard** [comment]
Can confirm the same behaviour here on Mac OSX. Additionally, many of my logs are at 0ms.  (I may be having unrelated issues due to ng-language-service running out of memory when scanning node_modules). For example:


    Angular Language Service: 5.0.0-beta.5
    TypeScript: 2.4.2
    getTemplateRefrences: 200ms
    Search path: {Project_Path}/node_modules/html-minifier/node_modules/uglify-js/tools
    Config file name: {Project_Path}/tsconfig.json
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms
    getTemplateRefrences: 0ms

**Comment by snewell92** [comment]
Related to #187

**Comment by shaneparsons** [comment]
I'm getting similar behaviour, but different output... Restarting vscode seems to fix it, but only temporarily. Is this related, or should I open a new issue?

```
Search path: /Users/Shane/Code/app/src/pages/supplies
Config file name: /Users/Shane/Code/app/tsconfig.json
SERVER ERROR: Debug Failure. 
Error: Debug Failure. 
    at tryReuseStructureFromOldProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:69547:22)
    at Object.createProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:69310:34)
    at synchronizeHostData (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:89172:33)
    at Object.getProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:89273:13)
    at Project.updateGraph (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:814:61)
    at Project.finishGraph (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:807:14)
    at ProjectService.updateConfiguredProject (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1717:25)
    at ProjectService.openOrUpdateConfiguredProjectForFile (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1501:22)
    at ProjectService.openClientFile (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1404:23)
    at /Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/documents.js:135:47
[Error - 11:22:29] Notification handler 'textDocument/didOpen' failed with message: Debug Failure. 
Search path: /Users/Shane/Code/app/src/pages/supplies
Config file name: /Users/Shane/Code/app/tsconfig.json
SERVER ERROR: Debug Failure. 
Error: Debug Failure. 
    at tryReuseStructureFromOldProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:69547:22)
    at Object.createProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:69310:34)
    at synchronizeHostData (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:89172:33)
    at Object.getProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:89273:13)
    at Project.updateGraph (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:814:61)
    at Project.finishGraph (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:807:14)
    at ProjectService.updateConfiguredProject (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1717:25)
    at ProjectService.openOrUpdateConfiguredProjectForFile (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1501:22)
    at ProjectService.openClientFile (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1404:23)
    at /Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/documents.js:135:47
[Error - 11:22:29] Notification handler 'textDocument/didOpen' failed with message: Debug Failure. 
Search path: /Users/Shane/Code/app/src/pages/supplies
Config file name: /Users/Shane/Code/app/tsconfig.json
SERVER ERROR: Debug Failure. 
Error: Debug Failure.
    at tryReuseStructureFromOldProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:69547:22)
    at Object.createProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:69310:34)
    at synchronizeHostData (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:89172:33)
    at Object.getProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:89273:13)
    at Project.updateGraph (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:814:61)
    at Project.finishGraph (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:807:14)
    at ProjectService.updateConfiguredProject (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1717:25)
    at ProjectService.openOrUpdateConfiguredProjectForFile (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1501:22)
    at ProjectService.openClientFile (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1404:23)
    at /Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/documents.js:135:47
[Error - 11:22:30] Notification handler 'textDocument/didOpen' failed with message: Debug Failure. 
Search path: /Users/Shane/Code/app/src/pages/supplies
Config file name: /Users/Shane/Code/app/tsconfig.json
SERVER ERROR: Debug Failure. 
Error: Debug Failure. 
    at tryReuseStructureFromOldProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:69547:22)
    at Object.createProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:69310:34)
    at synchronizeHostData (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:89172:33)
    at Object.getProgram (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/node_modules/typescript/lib/typescript.js:89273:13)
    at Project.updateGraph (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:814:61)
    at Project.finishGraph (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:807:14)
    at ProjectService.updateConfiguredProject (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1717:25)
    at ProjectService.openOrUpdateConfiguredProjectForFile (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1501:22)
    at ProjectService.openClientFile (/Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/editorServices.js:1404:23)
    at /Users/Shane/.vscode/extensions/Angular.ng-template-0.1.7/server/documents.js:135:47
[Error - 11:22:30] Notification handler 'textDocument/didOpen' failed with message: Debug Failure. 
```

**Comment by kyliau** [comment]
This has been fixed in v0.900.0 - debugging statements like these are no longer printed to the console.
